hack/packages/check-sample-values-package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# this shell script checks the existence os a sample values file for a Tanzu package and verifies if ytt templates are generated.
+
+# set this value to your package name
+PACKAGE=$1
+
+# set this value to the version of your package
+VERSION=$2
+
+if [ -z "$PACKAGE" ]
+then
+  echo "check sample values for package failed. must set NAME"
+  exit 2
+fi
+
+if [ -z "$VERSION" ]
+then
+  echo "check sample values for package failed. must set VERSION"
+  exit 2
+fi
+
+ROOT_DIR="addons/packages"
+BUNDLE_DIR="bundle"
+CONFIG_DIR="config"
```

Comment:
These definitions for ```BUNDLE_DIR="bundle"``` and ```CONFIG_DIR="config"``` seem redundant.

Reply:
Fixed

Reply:
@shivaani0505 I think you can remove these for ```verify-openapischema-for-package.sh``` as well

Reply:
Fixed